### PR TITLE
display map in map section of homepage

### PR DESCRIPTION
**Pull request by Omarzintan**

### CODE REVIEW

**Review by ellisvivian — DISMISSED** [review]

File: src/main/webapp/mapScript.js [hunk 1]
```diff
@@ -2,6 +2,33 @@
 function initMap() {
   const map = new google.maps.Map(document.getElementById("map"), {
     center: { lat: -34.937, lng: 150.644 },
-    zoom: 8
+    zoom: 14
   })
+
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function(position) {
```
Comment (ellisvivian) [inline src/main/webapp/mapScript.js:10]: Maybe just add a couple comments to clarify what exactly you are doing. For example, at first I didn't understand what "if (navigator.geolocation)" was checking until below I saw the comment for the else statement.

Comment (ellisvivian) [reply src/main/webapp/mapScript.js:10]: And not specifically this line, but in general.

Comment (lexidelorey) [reply src/main/webapp/mapScript.js:10]: +1

Comment (Omarzintan) [reply src/main/webapp/mapScript.js:10]: Okay, will do. Thanks

**Review by lexidelorey — APPROVED** [review]

**Pull request merged**